src/app/static/src/app/types.ts

```diff
@@ -203,7 +203,7 @@ export interface ReviewInputDataset {
     warnings: Warning[]
 }
 
-export interface StepWarnings {
+export interface StepWarnings extends Dict<Warning[]> {
```

Comment:
`Stepper.vue` was complaining about this so added an extends to tell it that this is in fact a `Dict<Warning[]>` even though really typescript shouldve been able to figure this out